docs/source/docs/contributing/building-photon.md

```diff
@@ -243,19 +243,9 @@ The program will wait for the VSCode debugger to attach before proceeding.
 
 ### Running examples
 
-You can run one of the many built in examples straight from the command line, too! They contain a fully featured robot project, and some include simulation support. The projects can be found inside the photonlib-java-examples and photonlib-cpp-examples subdirectories, respectively. The projects currently available include:
-
-- photonlib-java-examples:
```

Comment:
Should we add these back?

Reply:
Huh didn't fully mean to take that out. I just didn't want a specific list of examples (maintain same info in two spots). This should also have the "how to run python" stuff too.